aws/assets/license.txt

```diff
@@ -0,0 +1,339 @@
+                    GNU GENERAL PUBLIC LICENSE
+                       Version 2, June 1991
```

Comment:
we may have to change the license for 5.6

Reply:
new license added